File: season2/2주차/주사위세개/정성헌.java
import java.util.*;
public class Main {
    public static void main(String[] args) {

        Scanner sc=new  Scanner(System.in);

        int []arr=new int[3];
        int maxN=0;
        int answer=1;
        for(int i=0; i<3; i++)
        {
            arr[i]= sc.nextInt();
        }

        if(arr[0]==arr[1])
        {
            if(arr[1]==arr[2])
            {
                answer=10000+arr[0]*1000;
                System.out.println(answer);
            }
            else
            {
                answer=1000+arr[0]*100;
                System.out.println(answer);
            }
        }
        else
        {
           if(arr[0]==arr[2])
           {
               answer=1000+arr[0]*100;
               System.out.println(answer);
           }
           else if(arr[1]==arr[2])
           {
               answer=1000+arr[1]*100;
               System.out.println(answer);
           }
           else
           {
               for(int i=0; i<3; i++)
               {
                  maxN=Math.max(maxN,arr[i]);
               }
               answer=maxN*100;
               System.out.println(answer);
           }
        }


    }
}

File: season3/5주차/주식/정성헌.cpp
#include<bits/stdc++.h>
using namespace std;
long long T, N, arr[1000001],Max,answer;
int main()
{
	cin >> T;

	while (T--)
	{
		cin >> N;
		answer = 0;
		Max = 0;
		for (int i = 0; i < N; i++)
		{
			cin >> arr[i];
		}

		for (int i = N - 1; i >= 0; i--)
		{
			Max = max(Max, arr[i]);
			answer += (Max - arr[i]);
		}
		cout << answer << "\n";
		
	}
}